content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -90,7 +90,7 @@ disruptions, if any, to expect.
 
 ## Pod disruption budgets
 
-{{< feature-state for_k8s_version="v1.5" state="beta" >}}
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
```

Comment:
Can you make sure to make changes to the feature gate page, remove `PodDisruptionBudget` from [Feature gates for Alpha or Beta features](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features) to [Feature gates for graduated or deprecated features](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-graduated-or-deprecated-features). When removing from the alpha/beta table, please make sure to remove include the alpha/beta history in the GA table. Thanks!

Reply:
Updated